src/super_gradients/recipes/coco2017_pose_dekr_w32_no_dc.yaml

```diff
@@ -55,6 +55,9 @@ ckpt_root_dir:
 train_dataloader: coco2017_pose_train
 val_dataloader: coco2017_pose_val
 
+arch_params:
```

Comment:
How is that related?

Reply:
Not related.
I ran our regression tests, and this one was crashing because of missing arch_params.num_classes.
Thouht Itd be a good idea to fix this now and not forget about it later.